### PR TITLE
string.Format: fixed How arguments are formatted section

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes dotnet/docs#5151

### CODE REVIEW

File: xml/System/String.xml [hunk 1]
```diff
@@ -4793,15 +4793,17 @@ In .NET Core, sorting and comparison operations are based on [Version 8.0.0 of t
...
   
 -   If the argument implements the <xref:System.IFormattable> interface, its <xref:System.IFormattable.ToString%2A?displayProperty=nameWithType> implementation is called.  
   
--   The argument's parameterless `ToString` method, which is either overridden or inherited from the <xref:System.Object> class, is called.  
+-   The argument's parameterless `ToString` method, which is either overridden or inherited from a base class, is called.
```
Comment (rpetrusha) [inline xml/System/String.xml]: again a nit unrelated to your PR: is overridden is wrong; it should be "which either overrides or inherits from a base class implementation".

**Comment by pkulikov** [comment]
@rpetrusha thank you for review; I've addressed your feedback. You've mentioned *one comment about your PR*, though I see only two corrections unrelated to PR.

File: xml/System/String.xml [hunk 2]
```diff
@@ -4793,15 +4793,17 @@ In .NET Core, sorting and comparison operations are based on [Version 8.0.0 of t
...
+-   If you call the <xref:System.String.Format%28System.IFormatProvider%2CSystem.String%2CSystem.Object%5B%5D%29> overload and the `provider` object's <xref:System.IFormatProvider.GetFormat%2A?displayProperty=nameWithType> implementation returns non-null <xref:System.ICustomFormatter> implementation, the argument is passed to its <xref:System.ICustomFormatter.Format%28System.String%2CSystem.Object%2CSystem.IFormatProvider%29?displayProperty=nameWithType> method. If the format item includes a *formatString* argument, it is passed as the first argument to the method. If the <xref:System.ICustomFormatter> implementation is available and produces non-null string, that string is returned as the string representation of the argument; otherwise, the next step executes.
```
Comment (rpetrusha) [inline xml/System/String.xml]: There are two occurrences of "non-null" here; both should be preceded by a ("a non-null")

**Comment by pkulikov** [comment]
@rpetrusha thanks! Fixed!

**Comment by rpetrusha** [comment]
Closing and reopening to force a new license/cla check.

**Comment by BillWagner** [comment]
merging based on @rpetrusha approval.

**Pull request merged**